docs/modules/getting-started/pages/metrics.adoc

```diff
@@ -0,0 +1,69 @@
+= AeroGear Services Metrics SDK
+
+This document explains how to use and integrate the AeroGear Metrics SDK on a Cordova application.
+
+=== Install Dependencies
+
+Install the necessary dependencies in the app by running the following commands in your terminal:
+
+[source.bash]
+npm install @aerogear/app @aerogear/core
+cordova plugin add @aerogear/cordova-plugin-aerogear-metrics
```

Comment:
This is only when you have metrics server so I will not consider this an generic step.

Reply:
What's the point then, if you don't have metrics, you wouldn't even consider looking at this documentation.

Reply:
Point was to build getting started. We do not need metrics getting started etc.